### ISSUE TITLE
Threatpinch Extension was pulled from the Google Extension Store

### ISSUE THREAD

**Issue by CyberKaizen** [cloudtracer/ThreatPinchLookup#89]
ThreatPinch got pulled from the Google Web Store over a Web Store Policy?
<img width="672" alt="image" src="https://user-images.githubusercontent.com/78124503/197849358-75293b60-d09f-4ecf-b9da-f22cad2d75ea.png">

**Comment by H3liumBoy** [comment]
Yeah, sad indeed. I'd hoped to jump ship and switch to Firefox with the pending manifestv3. The only thing keeping me on Chrome was this add-on (as the Firefox workaround seems to be removed a while ago). Guess that issue is fixed now as well :-( 

Hope it gets reinstated, but the project does not seem to be alive anymore.

**Comment by cloudtracer** [comment]
Thanks for the report, 

There was an issue with the ThreatPinch website - which hosts the privacy policy - and the plugin was pulled due to the privacy policy being unavailable.  The ThreatPinch Lookup plugin should be available in the Chrome store again and the lite version is currently in the review stage and should be available soon.
